apps/wing/src/analytics/storage.ts

```diff
@@ -0,0 +1,53 @@
+import { existsSync, readFileSync, writeFileSync } from "fs";
+import { AnalyticEvent, AnalyticsReport } from "./event";
+import { randomBytes } from "crypto";
+
+const WING_ANALYTICS_FILE = 'wing-analytics.json';
```

Comment:
Just temporary constant file for testing, will change as PR transitions to Ready for Review